index.html

```diff
@@ -159,71 +159,90 @@ <h2>
-        that the UA has either obtained <a>expressed permission</a>, and/or
-        <a href="#askforgiveness">asked for forgiveness</a>, and/or ensured a
-        <a>prearranged trust relationship</a> exists.
+        Within this section, <var>descriptor</var> is an instance of the
+        <a>permission descriptor type</a> of the <code>"wake-lock"</code>
+        <a>powerful feature</a>.
       </p>
       <p>
-        The
-        <a href="https://www.w3.org/TR/permissions/#dictdef-permissiondescriptor">
-        <dfn>wake lock permission name</dfn></a> is
-        <a href="https://github.com/w3c/permissions/issues/47">defined</a> as
-        <code>"wake-lock"</code>.
+        To <dfn>block a permission</dfn>, run these steps:
       </p>
+      <ol>
+        <li>
+          Let <var>descriptor</var> be the <a>permission descriptor type</a>
+          instance to be blocked.
+        </li>
+        <li>
+          Set <var>descriptor</var>'s <a>permission state</a> to <code>"denied"</code>.
+        </li>
+        <li>
+          Let <var>record</var> be the <a>platform wake lock</a>'s
+          <a>state record</a> associated with <var>descriptor</var>'s
+          <a data-lt="WakeLockPermissionDescriptor.type">type</a> member.
+        </li>
+        <li>
+          <a>For each</a> <var>lock</var> in <var>record</var>.<a>[[\InstanceList]]</a>:
+          <ol>
+            <li>
+              Run <a>release a wake lock</a> on <var>lock</var>.
+            </li>
+          </ol>
+        </li>
+      </ol>
       <p>
-        To <dfn>obtain permission</dfn>, run these steps <a>in parallel</a>:
+        To <dfn>obtain permission</dfn> for <a>wake lock type</a>
+        <var>type</var>, run these steps <a>in parallel</a>:
       </p>
       <ol>
         <li>
-          If there is a <a>prearranged trust relationship</a>,
-          return <code>true</code>.
+          Let <var>permissionDesc</var> be the following
+          <a>permission descriptor type</a>:
+          <pre>
+            {
+              <a data-lt="permission name">name</a>: "wake-lock",
+              type: <var>type</var>
+            }
+          </pre>
         </li>
         <li>
           Let <var>permission</var> be the result of running
-          <a>query a permission</a> for the
-          <a>wake lock permission name</a>.
-        </li>
-        <li>
-          If <var>permission</var> is <code><a>"granted"</a></code>
-          (i.e. an <a>expressed permission</a> has been granted
-          to the <a>origin</a> and <a>global object</a> using
-          the Permissions API [[!PERMISSIONS]]), return <code>true</code>.
+          <a>query a permission</a> with <var>permissionDesc</var>.
         </li>
         <li>
           If <var>permission</var> is <code><a>"denied"</a></code>,
           return <code>false</code>.
         </li>
+        <li>
+          If <var>permission</var> is <code><a>"granted"</a></code>,
```

Comment:
```suggestion
          If <var>permission</var> is <a>"granted"</a>,
```

Definitions wrapped in `<code>` will automatically get a code element around any `<a>`